Airports/EIDW/Runway.txt

```diff
@@ -1,2 +1,3 @@
-10R  28L  098 278 N053.25.20.750 W006.17.24.270 N053.25.12.940 W006.15.02.080
-16  34  159 339 N053.26.13.160 W006.15.43.120 N053.25.11.660 W006.14.58.540
\ No newline at end of file
+10R  28L  095 275 N053.25.20.750 W006.17.24.270 N053.25.12.940 W006.15.02.080
+10L  28R  095 275 N053.26.13.790 W006.16.50.220 N053.26.05.390 W006.14.17.600
```

Comment:
```suggestion
10L 28R 098 278 N053.26.13.790 W006.16.50.220 N053.26.06.730 W006.14.41.870
```
Double space is used when no left/right indicator. Headings should be magnetic.
Seems there was also an error in the threshold co-ord of 28R (used 10L Geoid co-ord instead I think).